README.md

```diff
@@ -18,48 +15,61 @@ The purpose of this assignment is to reinforce the following concepts:
 
 ### Regarding the Word "Media"
 
-The Rails inflector considers "media" to be the plural of "medium", which is not really what we mean here. You may want to choose a different word to represent "a book, movie or album" internally. The instructor-proved example site uses the word "work".
+Our project is called Media Ranker, and revolves around voting on media. You may suspect that "A Single Piece of Media," or "Media," is going to be a model.
+
+However, the Rails inflector considers "media" to be the plural of "medium", which is not really what we mean here. You may want to choose a different word to represent "a book, movie or album" internally. The instructor-proved example site uses the word "work".
+
+### What We've Included
 
-## Project Requirements
+Inside of the `db` folder, we have included two files:
 
-### Core Requirements
+- `media-seeds.csv`
+- `generate_starter_data.rb`
 
-Regardless of how you choose to implement this project or how much of it gets done, you should exhibit
+You are encouraged to build your own `seeds.rb` script using the `media-seeds.csv` file, so that you can run `rails db:seed`. You are encouraged to reference other seeds scripts you may have!
+
+The file `generate_starter_data.rb` is a script to help generate a new `media-seeds.csv` file. There are comments in the file about its use. You are not required to use this file.
+
+## Non-Functional Requirements
+
+Regardless of how you choose to implement this project or how it progresses over time, your workflow should exhibit:
 
 - Squeaky-clean **git hygiene**, including
   - A fresh branch for each new feature
   - Regular commits
   - Descriptive commit messages
 - Fanatical devotion to **test-driven development**
-  - Pseudocode first, then write the tests, then write code to make them pass
+  - Pseudocode first, then write the tests, then write code to make them pass (with the exception of when we encourage otherwise)
 - Steadfast adherence to **agile development practices**
   - User stories should be listed and prioritized using a Trello board
   - The finished application should be deployed to Heroku (deploy early, deploy often)
 - Unrelenting use of **semantic HTML**
 
-### Baseline
-
-We will begin with some in-class work, exploring the site and pondering implementation details. Before you start writing _any_ code, you should:
-
-- Explore the existing Media Ranker site to become familiar with the necessary functionality
-- Create a Trello board to manage user stories
-- Create an ERD for the models
-
-Then, once you have a solid plan for how to structure your project:
-
-- Fork and clone the repo
-- Use `rails new .` to generate a new Rails project in the cloned directory
-  - Verify that the changes we've made to Rails' defaults (postgres as the DB, spec-style testing) have been applied
-- `git add .` and `git commit -m "Initial Rails setup"`
+## Functional Requirements
 
 ### Wave 1
```

Comment:
Just above we recommend writing tests then writing the code to make them pass, but here we recommend writing code then writing tests.

Reply:
nevermind, I see that you wrote "(with the exception of when we encourage otherwise)" .. I'm obviously still learning how to read changed files fluently.